src/components/Quiz/QuizPlayQuestionStrip.vue

```diff
@@ -66,26 +71,32 @@ export default {
 			countdown: null,
 			questionUrl: '',
 			text: '',
-			answer: null,
+			points: null,
 			questionNo: null,
 			runId: null,
 			click: 0,
 		};
 	},
 	mounted() {
-		this.runId = quizService.getRunId();
+		this.runId = this.$route.params.runId;
 		this.loadQuiz(this.$route.params.id);
 		this.countdown = this.question.duration;
 	},
 	methods: {
 		submitAnswer(selectedOption) {
 			this.click = 1;
+			if (selectedOption == this.question.answer) {
+				this.points = 100;
```

Comment:
the frontend should not receive the correct answer from api for players. Also points calculation should be done on the api side